core/templates/domain/collection/CollectionObjectFactorySpec.ts

```diff
@@ -13,29 +13,18 @@
 // limitations under the License.
 
 /**
- * @fileoverview Tests for CollectionObjectFactory.
+ * @fileoverview Tests for collection-model.
```

Comment:
```suggestion
 * @fileoverview Tests for CollectionModel.
```

So that it matches other files.